lib/Ray/sources/Drawables/Image.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+** EPITECH PROJECT, 2021
+** Bomberman
+** File description:
+** Image
+*/
+
+#ifndef IMAGE_HPP_
```

Comment:
Why is there an image header on this PR?

Reply:
ir's a merge issue

Reply:
fixed it